src/lib/terrain/patch/factory/cube.ts

```diff
@@ -0,0 +1,147 @@
+import * as THREE from "three";
```

Comment:
Might be better to import only what's needed to avoid packaging the entire threejs ?